src/lib/components/healthselectorv2/HealthSelector.component.js

```diff
@@ -0,0 +1,233 @@
+//@flow
+import React from 'react';
+import styled from 'styled-components';
+import Select, { Option } from '../selectv2/Selectv2.component';
+import Icon from '../icon/Icon.component';
+import * as defaultTheme from '../../style/theme';
+
+type optionType = 'all' | 'healthy' | 'warning' | 'critical' | 'unknown';
+type optionValue = {
+  icon?: React.Node,
+  label?: string,
+  shortLabel?: string,
+  value?: string,
+  disabled?: boolean,
+};
+
+type optionsConfiguration = { [optionType]: optionValue };
+
+type Props = {
+  optionsConfiguration?: optionsConfiguration,
+  label?: String,
```

Comment:
```suggestion
  label?: string,
```
Primitive types in flow are not capitalised